app.py

```diff
@@ -52,7 +63,9 @@ def get_cases_from_db(num_of_cases_to_fetch=int(Config.CASES_TO_FETCH)):
     SELECT c.case_id, c.case_ref, c.case_type, c.address_level, c.region, c.treatment_code, u.uac, u.qid
     FROM casev2.cases c, casev2.uac_qid_link u
     WHERE u.caze_case_id = c.case_id
-    LIMIT {num_of_cases_to_fetch};'''
+    ORDER BY c.case_ref
```

Comment:
Should this be configurable to select the range of cases it's selecting from? Otherwise running these in parallel won't it select the same cases each time?

Reply:
Added Offset to allow for different ranges